moto/mediastore/models.py

```diff
@@ -59,18 +61,23 @@ def put_lifecycle_policy(self, container_name, lifecycle_policy):
         if container_name not in self._containers:
             raise ResourceNotFoundException()
         self._containers[container_name].lifecycle_policy = lifecycle_policy
-        # print(lifecycle_policy)
-        # print(container_name)
         return {}
 
-    
+    def get_lifecycle_policy(self, container_name):
+        if container_name not in self._containers:
+            raise ResourceNotFoundException()
+        lifecycle_policy = self._containers[container_name].lifecycle_policy
```

Comment:
What if there _is_ such a container, but it doesn't have a lifecycle policy yet?